.docker/docker_run_git.sh

```diff
@@ -99,6 +99,11 @@ set -e
 if [ $PS_DEV_MODE -ne 1 ]; then
   echo "\n* Disabling DEV mode ...";
   sed -i -e "s/define('_PS_MODE_DEV_', true);/define('_PS_MODE_DEV_',\ false);/g" /var/www/html/config/defines.inc.php
+else
+  echo "\n* Enabling DEV mode ...";
+  sed -i -e "s/define('_PS_MODE_DEV_', false);/define('_PS_MODE_DEV_',\ true);/g" /var/www/html/config/defines.inc.php
```

Comment:
is this really needed ? it's supposed to be set to true by default

Reply:
ah or is it something specific to UI tests where it's set to false and must be set to true ?

Reply:
Yeah it's to test the classic build, since the tests are based on the export of the CreateRelease script which automatically sets the dev mode to false this part is needed to reset it back to true (for debug purposes)